src/samples/article-vor/v4/00569.json

```diff
@@ -0,0 +1,7679 @@
+{
+    "status": "vor",
+    "id": "00569",
+    "version": 1,
+    "type": "research-article",
+    "doi": "10.7554/eLife.00569",
+    "authorLine": "Ayelet Voskoboynik et al.",
+    "title": "The genome sequence of the colonial chordate, <i>Botryllus schlosseri</i>",
+    "published": "2013-07-02T00:00:00Z",
+    "stage": "published",
+    "statusDate": "2013-07-02T00:00:00Z",
+    "volume": 2,
+    "elocationId": "e00569",
+    "pdf": "https://elife-publishing-cdn.s3.amazonaws.com/00569/elife-00569-v1.pdf",
+    "figuresPdf": "https://elife-publishing-cdn.s3.amazonaws.com/00569/elife-00569-figures-v1.pdf",
+    "subjects": [
+        {
+            "id": "developmental-biology-stem-cells",
+            "name": "Developmental Biology and Stem Cells"
+        },
+        {
+            "id": "genomics-evolutionary-biology",
+            "name": "Genomics and Evolutionary Biology"
+        }
+    ],
+    "abstract": {
```

Comment:
should we have an article with a structured abstract as an example of a valid v4 article?

Reply:
> should we have an article with a structured abstract as an example of a valid v4 article?

I've added one now.